pkg/archive/untar_nested.go

```diff
@@ -106,6 +106,10 @@ func untarNested(r io.Reader, dir string) (err error) {
 				return err
 			}
 			madeDir[abs] = true
+		case os.ModeSymlink != 0:
```

Comment:
Did you run into a use-case that failed, and required this addition?

Is it in the upstream code?

https://cs.opensource.google/go/x/build/+/ef7e9155:internal/untar/untar.go;l=27

Reply:
The nodejs binary contains Symlinks for `npm` and `npx` which did not worked, when used the existing untar_nested.

Reply:
Thanks

Reply:
I ran into this with the github-actions runner too, which also bundles npm. :+1: